docs/docs/code/python/README.md

```diff
@@ -144,6 +144,33 @@ def handler(pd: "pipedream"):
   r = requests.post(url='https://api.imgur.com/3/image', files=files)
 ```
 
+
+
+
+## Returning HTTP responses
+
+You can return HTTP responses from [HTTP-triggered workflows](/workflows/steps/triggers/#http) using the `pd.respond()` method:
+
+```python
+def handler(pd: 'pipedream'):
+  pd.respond({
+    'status': 200,
+    'body': {
+      'message': 'Everything is ok'
+    }
+  })
+```
+
+:::warning
+
+Unlike the [Node.js equivalent](https://pipedream.com/docs/workflows/steps/triggers/#http-responses) `$.respond`, the Python `pd.respond` helper does not yet support responding with Streams.
```

Comment:
Maybe clarify what you can return (since a new Python user might not know the valid types of return values), in addition to the fact that stream responses aren't supported, as you note.